packages/block-library/src/term-description/index.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Server-side rendering of the `core/term-description` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/term-description` block on the server.
+ *
+ * @param array    $attributes Block attributes.
+ * @param string   $content    Block default content.
+ * @param WP_Block $block      Block instance.
+ * @return string Returns the filtered post content of the current post.
+ */
+function render_block_core_term_description( $attributes, $content, $block ) {
+
+	$description = false;
```

Comment:
I'm wondering if `is_archive` could be enough or correct 😄   🤔 .  `term_description` will not return anything if no term found.

Reply:
I used `is_category() || is_tag() || is_tax()` instead. Archives can be for dates and other things, so limiting it to taxonomies seems safer :)